Doc/c-api/long.rst

```diff
@@ -571,6 +571,42 @@ distinguished from a number.  Use :c:func:`PyErr_Occurred` to disambiguate.
    .. versionadded:: 3.14
 
 
+.. c:function:: int PyLong_IsPositive(PyObject *obj)
+
+   Check if the integer object *obj* is positive.
+
+   On success, return 1 if *obj* is positive, and 0 otherwise.
+
+   On failure, return -1 with an exception set.  This function always succeeds
+   if *obj* is a :c:type:`PyLongObject` or its subtype.
```

Comment:
```suggestion
   Check if the integer object *obj* is positive.

   If *obj* is an instance of :c:type:`PyLongObject` or its subtype,
   return ``1`` when it's positive and ``0`` otherwise.  Else set an
   exception and return ``-1``.
```
@serhiy-storchaka ?

Reply:
For this, maybe we should wait for serhiy's reply :)

Reply:
I am fine with this.